_data/events.yml

```
@@ -63,7 +63,7 @@ scala_lunches:
       time: "12:00pm"
       place: BC 410, EPFL
       speaker: "Dimi Racordon"
-      title: # to be announced
+      title: # "Demystifying type inference: a quick look dive into the implementation of a typer"
```

```suggestion
      title: "Demystifying type inference: a quick look dive into the implementation of a typer"
```
The `#` means "don't look at what's on the right, so we need to remove it.